packages/mobile/src/store/offline-downloads/sagas/watchRemoveOfflineItems.ts

```diff
@@ -24,23 +29,36 @@ function* deleteItemsFromDisk(action: RemoveOfflineEntriesAction) {
   const trackStatus = yield* select(getOfflineTrackStatus)
   const collectionStatus = yield* select(getOfflineCollectionsStatus)
 
-  for (const item of items) {
+  yield* all(
+    items.map((item) =>
+      call(removeItemFromDisk, item, trackStatus, collectionStatus)
+    )
+  )
+}
+
+function* removeItemFromDisk(
+  item: OfflineEntry,
+  trackStatus: TrackStatus,
+  collectionStatus: CollectionStatus
+) {
+  if (item.type === 'collection' && !collectionStatus[item.id]) {
+    track(make({ eventName: EventNames.OFFLINE_MODE_REMOVE_ITEM, ...item }))
```

Comment:
this feels like it could be a lot of events. amplitude react native will batch them, with configurable concurrency but maybe we should do one event w more metadata?

Reply:
I def agree. Ik not too sure why this data would be useful even... Maybe can chat with Andrew about it

Reply:
Hmm yeah I don't think we need to track that. If we do, we should do the count of times the toggle is turned off and/or the total number of items to be deleted.